web_app/analisis/views.py

```diff
@@ -63,17 +63,38 @@ def list(self, request):
         serializer = AnswerSerializer(queryset, many=True)
         return Response(serializer.data)
 
+
+def add_id_to_in_progress_ids(_id):    
+    from django.core.cache import cache
+    cache.set('IN_PROGRESS_IDS', [_id] + cache.get('IN_PROGRESS_IDS',[]), timeout=36000)# 1 hour
+    print('Add id to in progress ids:', cache.get('IN_PROGRESS_IDS',[]))
+    return None
+
+def remove_id_from_in_progress_ids(input_id):
+    from django.core.cache import cache
+    cache.set('IN_PROGRESS_IDS', [_id for _id in cache.get('IN_PROGRESS_IDS',[]) if _id != input_id ], timeout=36000)
```

Comment:
We could set timeout value as a constant. But that is optional.

Reply:
Added CACHE_TIMEOUT constant in settings file